test/test_overrides.py

```diff
@@ -275,6 +284,17 @@ def sub_diagonal_foo(a, b, c=None):
 
 # The dispatch table for SubDiagonalTensor's __torch_function__ implementation.
 HANDLED_FUNCTIONS_TENSOR_LIKE = {}
+HANDLED_FUNCTIONS_NAMESPACES = {}
```

Comment:
What is this? `HANDLED_FUNCTIONS_NAMESPACES.getattr(func_name)` gives you the func. Is this for testing? The tests have two separate cases: first is testing for torch.Tensor subclasses, and the other is TensorLikes. Tensor methods only apply to subclasses, they don't apply to other TensorLikes.